dipy/segment/clustering.py

```diff
@@ -508,3 +508,339 @@ def cluster(self, streamlines, ordering=None):
 
         cluster_map.refdata = streamlines
         return cluster_map
+
+
+class QuickBundlesOnline(Clustering):
+    r""" Clusters streamlines using QuickBundles [Garyfallidis12]_.
```

Comment:
This docstring needs to be updated to reflect the "online" aspect of that algorithm.